Sources/Ruby/RubyMockServer.swift

```diff
@@ -1,33 +1,33 @@
 import Foundation
-import Alamofire
 import BrightFutures
 
-open class RubyMockServer: MockServer {
-  open let url: String
-  open let port: Int
+open class RubyMockServer: NSObject, MockServer {
+  public let url: String
+  public let port: Int
+  public let allowInsecureCertificates: Bool
 
   open var baseUrl: String {
     return "\(url):\(port)"
   }
 
-  enum Router: URLRequestConvertible {
+    enum Router {
     static var baseURLString = "http://example.com"
 
-    case clean()
+    case clean
     case setup([String: Any])
-    case verify()
+    case verify
     case write([String: [String: String]])
 
-    var method: HTTPMethod {
+    var method: String {
```

Comment:
HTTPMethod.rawValue should be the only place where the mapping of these should occur.

Reply:
I think HTTPMethod was from Alamofire which I removed as a dependency here.
So what would be the todo here?